dev_tools/nbformat

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# Formats ipython notebooks with tensorflow-docs nbformat tool.
```

Comment:
Did you write this or take it from somewhere? 

If the former: ick bash. I guess it's ok here since you aren't doing looping (which is my rule of thumb for when a script definitely needs to be in a real programming langauge)

If the latter: note where you got it from so we can keep them in sync

Reply:
This is taken directly from Cirq: https://github.com/quantumlib/Cirq/blob/master/check/nbformat

In addition to indentation, metadata is also removed and probably a few other things - @lamberta may know more here.

Not sure why the diff is so large - may be because GH has trouble analyzing notebooks but I'll check. At least by quick glance the notebooks are very similar before/after formatting.

Reply:
Try passing `--indent=2` to nbfmt. The default is 4 but the quantum folks prefer 2

Reply:
@lamberta I think the collab default is 2 but the jupyter default is 1 and we prefer 1

@rmlarose the diff is messed up because of the indentation of the json in the ipynb files. Currently recirq has some with 2 and some with 1